### DISCUSSION TITLE
Connect to GoPro Bluetooth instructins (assets)

### DISCUSSION THREAD

**Discussion by sultanmyrza** [gopro/OpenGoPro#43] [General]
Hello,
I'm making ionic app to connect to GoPro.

GoPro official app (Go Pro quick) has instructins on how to connect to GoPro for each GoPro Device model here is the example of what I mean https://youtu.be/2Fpfx3Fvcss

## Question

1. Where can I get those animations? 
2. Can it be open sourced to include into my app?

**Comment by KonradIT** [comment]
You could unpack the GoPro APK and get the videos (under the `assets` folder) but those artworks are copyrighted by GoPro.

**Reply by sultanmyrza** [reply]
For now I would like to try @KonradIT solution.
@tcamise-gpsw @tcheema-gpsw But at the time of pulishing my app is there a legall way of getting those assets?
